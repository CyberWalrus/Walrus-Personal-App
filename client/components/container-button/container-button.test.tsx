import * as Enzyme from "enzyme";
import { shallow } from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson, { Json } from "enzyme-to-json";
import * as React from "react";
import ContainerButton from "./container-button";

Enzyme.configure({ adapter: new Adapter() });
it(`ContainerButton correctly render`, (): void => {
  const jestFunc = jest.fn();
  const tree: Json = toJson(
    shallow(<ContainerButton onButtonCLick={jestFunc} />),
  );
  expect(tree).toMatchSnapshot();
});
