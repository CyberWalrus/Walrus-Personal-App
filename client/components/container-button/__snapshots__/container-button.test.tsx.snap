// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerButton correctly render 1`] = `
<div
  className="container-btn"
>
  <ul
    className="container-btn__ul"
  >
    <li
      className="container-btn__item"
    >
      <a
        className="container-btn__btn"
        href="#"
        onClick={[MockFunction]}
      >
        J
      </a>
    </li>
    <li
      className="container-btn__item"
    >
      <a
        className="container-btn__btn"
        href="#"
        onClick={[MockFunction]}
      >
        k
      </a>
    </li>
    <li
      className="container-btn__item"
    >
      <a
        className="container-btn__btn"
        href="#"
        onClick={[MockFunction]}
      >
        s
      </a>
    </li>
    <li
      className="container-btn__item"
    >
      <a
        className="container-btn__btn"
        href="#"
        onClick={[MockFunction]}
      >
        M
      </a>
    </li>
  </ul>
</div>
`;
